### PR TITLE
new reviewcontroller and reviewhtml

### PR DESCRIPTION
**Pull request by tkarim23**
check it out

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -1,16 +1,39 @@
 class ReviewsController < ApplicationController
   def new
+    @reviews = Review.new
```
Comment (a5hna) [inline app/controllers/reviews_controller.rb]: should be @review

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -1,16 +1,39 @@
...
+    @scooter = Scooter.find(params[:scooter_id])
   end
 
   def create
+    @reviews = Review.new(review_params)
```
Comment (a5hna) [inline app/controllers/reviews_controller.rb]: again @review

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -1,16 +1,39 @@
...
+    @scooter = Scooter.find(params[:scooter_id])
+    @review.scooter = @scooter
+    if @reviews.save
```
Comment (a5hna) [inline app/controllers/reviews_controller.rb]: @review

File: app/controllers/reviews_controller.rb (hunk 2 above)
Comment (tkarim23) [reply app/controllers/reviews_controller.rb]: ok

File: app/controllers/reviews_controller.rb (hunk 3 above)
Comment (tkarim23) [reply app/controllers/reviews_controller.rb]: ok

**Pull request merged**